packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [92.0.0]
+
 ### Added
 
 - **BREAKING:** Add optional JWT token authentication to multi-chain accounts API calls ([#7165](https://github.com/MetaMask/core/pull/7165))
```

Comment:
If the token is optional why did you decide to reference this as a breaking change ?